DFC.ServiceTaxonomy.ApiFunction/CypherQueries/V1/GetAutoCompleteJobProfilesBySearchTermNoAnalyzer .json

```diff
@@ -0,0 +1,14 @@
+{
+  "query": "with $searchTerm as lowerlabel call db.index.fulltext.queryNodes(\"JobProfiles2\", \"skos__prefLabel:\"+lowerlabel + \"\") yield node, score return distinct node.skos__prefLabel limit $size",
```

Comment:
Is there a corresponding PR to create the index?
(The original indexes are [here](https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-editor/blob/master/GetJobProfiles/CypherCommandRecipes/CreateFullTextSearchIndexes.recipe.json).)

Reply:
Adding the pr shortly